app/move/steps/create.js

```diff
@@ -77,6 +77,11 @@ module.exports = {
       'date_custom',
     ],
   },
+  '/move-agreed': {
+    controller: Base,
+    pageTitle: 'moves::details.heading',
```

Comment:
Worth matching to the step name for consistency. Wonder if `/agreement-status` is a better name for the step too as `/move-agreed` suggests a positive response rather than what we're capturing.